xml/dhcp_vendor_class.xml

```diff
@@ -190,6 +190,18 @@
     <param pos="0" name="hw.device" value="Switch"/>
   </fingerprint>
 
+  <fingerprint pattern="^Aruba-Sensor(?:-([A-Z0-9]{6}))?$">
```

Comment:
Is `Aruba-Sensor` a valid value? If so, we should probably have an example for it.
```
$ echo "Aruba-Sensor" | bundle exec ./bin/recog_match xml/dhcp_vendor_class.xml
MATCH: {"matched"=>"HP Aruba UXI Sensor", "hw.vendor"=>"Aruba Networks", "hw.model"=>nil, "hw.device"=>"Sensor", "os.vendor"=>"Aruba Networks", "service.protocol"=>"dhcp", "fingerprint_db"=>"dhcp_vendor_class", "data"=>"Aruba-Sensor"}
```

Reply:
Yes, it's valid. See the 4th example below (line 198 in the file).

Reply:
I completely overlooked that example! 😅